website/project/model.py

```diff
@@ -1335,6 +1335,28 @@ def set_node_license(self, license_id, year, copyright_holders, auth, save=False
         if save:
             self.save()
 
+    def subscribe_user_to_notifications(self, user):
+        """ Update the notification settings for the creator or contributors
+
+        :param user: User to subscribe to notifications
+        """
+        from website.notifications import utils as notification_utils
```

Comment:
Because this is used only once here, can you do `from website.notifications.utils import to_subscription_key`?